hamilton/cli/logic.py

```diff
@@ -50,6 +52,8 @@ def get_git_reference(git_relative_path: Union[str, Path], git_reference: str) -
 
 def version_hamilton_functions(module: ModuleType) -> Dict[str, str]:
     """Hash the source code of Hamilton functions from a module"""
+    from hamilton import graph_utils
```

Comment:
why the inline import?

Reply:
it reduces the CLI startup time. For one-of commands it doesn't matter as much, but when using the Watcher, it can be pretty annoying.

For the back log, I want to profile the CLI and see how I can make things faster